app/models/upload.rb

```diff
@@ -72,6 +72,18 @@ def self.last_uploads_rows
     uploads.sort_by { |upload| upload.csv_type.downcase }
   end
 
+  def self.latest_uploads
```

Comment:
Can we get some specs for this method please?

Reply:
Created